<!DOCTYPE html>
<html>
    <head>
        <meta charset="utf-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
        <meta name="viewport" content="width=device-width">

        <title>MISO : Arrays tutorial</title>
        <meta name="description" content="An open-source LIMS for NGS sequencing centres">

        <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css" rel="stylesheet">
        <link rel="stylesheet" href="/miso-docs-oicr/styles/main.css">
    </head>
    <body>

        <div class="container">
            <div class="row">
                <div id="header" class="col-sm-12">
                    <h2><a class="brand" href="/miso-docs-oicr/"><img src="/miso-docs-oicr/pics/MISO.png"/>MISO</a>
    <small>An open-source LIMS for NGS sequencing centres</small>
</h2>
<span class="badge" style="position: relative;left:25px;bottom:20px;background:rgb(56,121,59);">OICR</span>

<span><a href="/miso-docs-oicr/plain-index">See the original flavour (plain sample) version of this walkthrough here</a></span>


                </div>
            </div>

            <div class="row">
                
                
                    <div id="content" class="col-sm-12">
                        <div class="page-header">
    <h2>Arrays tutorial
        
    </h2>
</div>

<div id="toc">
Table of Contents
<ol>
   <li><a href="#login">Logging In</a></li>
   <li><a href="#scan">Scan samples into your inbox</a></li>
   <li><a href="#arrays">Creating an Array</a></li>
   <li><a href="#arrayruns">Creating an Array Run</a></li>
</ol>
</div>

<p><a name="login"></a></p>

<h1 id="logging-in">1. Logging in</h1>

<p>Much like the old Geospiza LIMS, you need to log in to MISO LIMS in order to
make changes to any LIMS entities. 
Logging in lets MISO record any changes you make on entities you have access to.</p>

<p>If you were able to log in to Geospiza LIMS, you already have the correct
permissions and can proceed to logging in. If you are a new user, you will need
to contact <a href="&#109;&#097;&#105;&#108;&#116;&#111;:&#104;&#101;&#108;&#112;&#100;&#101;&#115;&#107;&#064;&#111;&#105;&#099;&#114;&#046;&#111;&#110;&#046;&#099;&#097;">&#104;&#101;&#108;&#112;&#100;&#101;&#115;&#107;&#064;&#111;&#105;&#099;&#114;&#046;&#111;&#110;&#046;&#099;&#097;</a> so that they put you into the appropriate
Active Directory group, <code class="highlighter-rouge">MISO_ROLE_INTERNAL</code>.</p>

<hr />

<p>Try to log in now:</p>

<ol>
  <li>Click on <a href="http://miso.gsi.oicr.on.ca" target="\_new">http://miso.gsi.oicr.on.ca</a>.</li>
  <li>Enter your
username (e.g. jdoe) and password and click the Login button.</li>
</ol>

<p>MISO uses the same username and password as your OICR email account.</p>

<p>If all goes well, you should see the MISO Dashboard and see a message at the
top right: &quot;Logged in as: <strong>jdoe</strong>&quot;.</p>

<p><a name="scan"></a></p>

<h1 id="scan-samples-into-your-inbox">2. Scan samples into your inbox</h1>

<p>First, scan the samples from the samples team into your inbox for further
work.</p>

<p>Boxes hold samples, libraries, and pools. They do not track reagents or
primers. Boxes are separated into <em>uses</em> for different kinds of storage
(<em>e.g.</em>, tissue samples versus extracted DNA), but there is no check
that items in a box match. Each box also has a <em>size</em> that includes the
physical dimensions of the box as well as whether the box is compatible
with the VisionMate scanner. Every <em>position</em> in the box is identified
by a standard row letter + column number format (<em>e.g.</em>, C05).</p>

<hr />

<p>In this exercise, we are going to take tissues/samples/libraries from the
previous team/collaborator by scanning them out of the previous box and into a
newly created &#39;inbox&#39;.</p>

<p>First, create a new box. This box will be your &#39;inbox&#39;.</p>

<ol>
  <li>From the navigation panel, choose <em>Boxes</em> and then <em>Add Box</em>.</li>
  <li>On the <em>Create Box</em> page, enter the information:</li>
  <li>Alias: A short name for the box. Record this name in your worksheet. <img src="pics/blue_pencil.png" /></li>
  <li><em>Description</em>: a human description of the box purpose  “gDNA ready for
    library prep for <project>”.</project></li>
  <li><em>Use</em>: The contents of the box. Select an appropriate use for your entities.</li>
  <li><em>Size</em>: Select <code class="highlighter-rouge">8 x 12 scannable</code>.</li>
  <li>Click <em>Save</em>.</li>
</ol>

<p>Upon clicking save, a graphic of the box will appear.</p>

<p><img src="pics/5-empty-box.png" id="figure" /></p>

<p>You can now fill the box with the tissues, stocks, aliquots, libraries, or
pools that were made in previous steps. Collect the appropriate matrix tubes/barcodes now.</p>

<ol>
  <li>Click on a position in the displayed <em>Contents</em> grid.</li>
  <li>Enter a matrix barcode into the box on the right and click <em>Lookup</em>.</li>
  <li>Click <em>Update Position</em>.
    <ul>
      <li>&#9888; The <em>Save</em> button at the top of the page does not work for
 individual positions, only for <em>Box Information</em>.</li>
    </ul>
  </li>
  <li>Repeat for as many samples/libraries/pools as you want.</li>
</ol>

<p>The table below the box diagram shows the position and information for the
currently selected sample. If you would like to see all of the samples in the
table, click the <em>List all Box Contents</em> button at the top right of the table.</p>

<p>In the lab, it is possible to use <em>Options</em> &gt; <em>Scan Box</em> to use the plate
scanner and update all positions at once, but that will not be covered in this
tutorial.</p>

<h2 id="using-boxes">Using Boxes</h2>

<p>Boxes can be found either from the Sample or Library page or the <em>Boxes</em> page.</p>

<ol>
  <li>On the <em>Sample</em> page, search for one of your sample or library aliases and click on the alias to load the page.</li>
  <li>The Box and position is listed under <em>Location</em> near the top of the <em>Sample
Information</em> section. Click on the link to go to the Box (e.g. <code class="highlighter-rouge">PROJ_OUTBOX,
A03</code>).</li>
</ol>

<p>Boxes can be used to store Samples, Libraries and Pools and one box can store all
three types.</p>

<p><a name="arrays"></a></p>

<p><a name="narrays" href="#" id="toplink">top</a></p>

<h1 id="arrays">3. Arrays</h1>

<p>An Array represents a chip which can have Samples added to it to be run on
an Array Scanner. Arrays can have different layouts depending on the chip model.</p>

<p>Note: Only Aliquot Samples may be added to an Array.</p>

<h1 id="create-an-array">3.1. Create an Array</h1>

<ol>
  <li>On the <em>Arrays</em> page, click the <em>Add</em> button</li>
  <li>Choose an <em>Array Model</em>. If there are no options in this list, ask your MISO
administrator to add some Array Models to MISO</li>
  <li>Enter a unique <em>Alias</em> and <em>Serial Number</em> for your Array</li>
  <li>Click <em>Save</em>. The array must be saved before samples can be added to it</li>
  <li>Add a Sample to the Array. The user interface is similar to Boxes:
    <ol>
      <li>Select an empty position on the array, which will appear as a grey circle</li>
      <li>In the <em>Search</em> box to the right, enter the Name, Alias, or Barcode of the
 Sample you wish to add</li>
      <li>Press <em>Enter</em> or click the <em>Lookup</em> button</li>
      <li>Your Sample should appear in the <em>Results</em> dropdown. Click <em>Update position</em>
 to add it to the array</li>
    </ol>
  </li>
  <li>Repeat for the other samples</li>
</ol>

<p><a name="arrayruns"></a></p>

<h1 id="create-an-array-run">4. Create an Array Run</h1>

<ol>
  <li>On the <em>Array Runs</em> page, click the <em>Add</em> button</li>
  <li>Choose an Array Scanner <em>Instrument</em>. If there are no options in this list, ask
your MISO administrator to add an Array Scanner to MISO</li>
  <li>Enter a unique <em>Alias</em> for your Array Run</li>
  <li>Enter the <em>Run Path</em> (directory) of the Array if available</li>
  <li>In the <em>Array Search</em> box, enter the Alias or Serial Number of your Array</li>
  <li>Press <em>Enter</em> or click the <em>Search</em> button</li>
  <li>Your Array should appear in the results dropdown. Click <em>Apply</em></li>
  <li>Select <em>Status</em>: <em>Running</em></li>
  <li>Enter today&#39;s date as the <em>Start Date</em> of the Array Run</li>
  <li>Click <em>Save</em></li>
</ol>

<table>
  <tbody>
    <tr>
      <td>&lt; <a href="4-0-sequencing">Sequencing tutorial</a></td>
      <td><a href="index">Home</a> &gt;</td>
    </tr>
  </tbody>
</table>


                    </div>
                
            </div>

            

            <div class="row">
                <div id="footer" class="col-sm-12">
                    Documentation for <a href="https://github.com/TGAC/miso-lims">MISO</a> developed by the <a href="http://labs.oicr.on.ca/genome-sequence-informatics/">Genome Sequence Informatics</a> group at <a href="http://oicr.on.ca">Ontario Institute for Cancer Research</a>


                </div>
            </div>
        </div>

        <script src="/miso-docs-oicr/styles/navigation.js"></script>

    </body>
</html>
