<!DOCTYPE html>
<html>
    <head>
        <meta charset="utf-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
        <meta name="viewport" content="width=device-width">

        <title>MISO : Samples Tutorial</title>
        <meta name="description" content="An open-source LIMS for NGS sequencing centres">

        <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css" rel="stylesheet">
        <link rel="stylesheet" href="/miso-docs-oicr/styles/main.css">
    </head>
    <body>

        <div class="container">
            <div class="row">
                <div id="header" class="col-sm-12">
                    <h2><a class="brand" href="/miso-docs-oicr/"><img src="/miso-docs-oicr/pics/MISO.png"/>MISO</a>
    <small>An open-source LIMS for NGS sequencing centres</small>
</h2>
<span class="badge" style="position: relative;left:25px;bottom:20px;background:rgb(56,121,59);">OICR</span>

<span><a href="/miso-docs-oicr/plain-index">See the original flavour (plain sample) version of this walkthrough here</a></span>


                </div>
            </div>

            <div class="row">
                
                
                    <div id="content" class="col-sm-12">
                        <div class="page-header">
    <h2>Samples Tutorial
        
    </h2>
</div>

<div id="toc">
Table of Contents
<ol>
    <li><a href="#login">Logging In</a></li>
    <li><a href="#receipt">Receiving Samples</a></li>
    <li><a href="#qcs">Adding Sample QCs</a></li>
    <li><a href="#boxes">Scanning samples into the outbox</a></li>
    <li><a href="#trouble">Troubleshooting</a></li>
</ol>
</div>

<div id="infobox">
Download the worksheet for this section here: <a href="2-0-samples-worksheet">Samples Worksheet</a>.
</div>

<p><a name="login"></a></p>

<h1 id="logging-in">1. Logging in</h1>

<p>You need to log in to MISO LIMS in order to make changes to any LIMS entities. 
Logging in lets MISO record any changes you make on entities you have access to.
If you are a new user and have not been given a username and password, you 
will first need to get a username and password by following the steps in the
<a href="plain-0-0-admin-tasks#users">Adding Users</a> section.</p>

<hr />

<p>Try to log in now:</p>

<ol>
  <li>Click on <a href="http://miso.gsi.oicr.on.ca" target="\_new">http://miso.gsi.oicr.on.ca</a>.</li>
  <li>Enter your
username (e.g. jdoe) and password and click the Login button.</li>
</ol>

<p>If all goes well, you should see the MISO Dashboard and see a message at the
top right: &quot;Logged in as: <strong>jdoe</strong>&quot;.</p>

<p><a name="receipt" href="#" id="toplink">top</a></p>

<h1 id="receiving-samples">2. Receiving samples</h1>

<p>A <em>Sample</em> contains information about the material upon which the sequencing
experiments are to be based. Samples can be used in any number of sequencing
<em>Experiments</em> in the form of a <em>Library</em> that is processed further into
single or pooled <em>Dilutions.</em></p>

<h2 id="receiving-a-sample">2.1 Receiving a Sample</h2>

<p>Samples are most easily entered into MISO using the bulk entry screen.</p>

<ol>
  <li>On the left hand menu under <em>Tracking</em>, click <em>Samples</em>.</li>
  <li>Click the <em>Create</em> button at the top left of the table.</li>
  <li>Select Sample Class <em>Tissue</em> and Quantity 1.</li>
  <li>Click <em>Create</em>.</li>
  <li>Enter or select the following in the table:
    <ol>
      <li>
        <p><em>Sample Name</em>: leave blank. This will be auto-generated after save.</p>
      </li>
      <li><em>Sample Alias</em>: enter a sample alias like <code class="highlighter-rouge">PRO1_S#_#</code>. Replace the <code class="highlighter-rouge">#</code>
symbol with numbers of your choosing.</li>
      <li>
        <p><em>Description</em>: <code class="highlighter-rouge">Wheat</code></p>
      </li>
      <li><em>Date of Receipt</em>: Select a date</li>
      <li><em>Sample Type</em>: select <code class="highlighter-rouge">GENOMIC</code> from the drop down.</li>
      <li><em>Scientific Name</em>: <code class="highlighter-rouge">Triticum aestivum</code>.</li>
    </ol>
  </li>
  <li>
    <p>Project: Select the project you created in the last exercise.</p>

    <ol>
      <li><em>QC Passed?</em>: <code class="highlighter-rouge">True</code></li>
    </ol>
  </li>
  <li>At the upper right hand side, click <em>Save</em>.</li>
</ol>

<p>Like in Excel, you can fill down a column by double-clicking the square at the lower 
right hand side of a selected cell. You can also click and drag to only fill in a 
certain number of cells.</p>

<table border="1"><tr><td>
<img src="pics/fill-down-1.png" />
</td><td>
<img src="pics/fill-down-2.png" />
</td></tr></table>

<p>Copy and pasting from Excel and Word is supported. Press Ctrl+V (Windows &amp; Linux) or 
Command+V (Mac) on your keyboard to paste.</p>

<h2 id="scanning-tissues-and-stocks-into-a-box">2.2 Scanning tissues and stocks into a Box</h2>

<p>In this section we will add the stocks and tissues you just received into your
inbox for further work.</p>

<p>Boxes hold samples, libraries, and pools. They do not track reagents or
primers. Boxes are separated into <em>uses</em> for different kinds of storage
(<em>e.g.</em>, tissue samples versus extracted DNA), but there is no check
that items in a box match. Each box also has a <em>size</em> that includes the
physical dimensions of the box as well as whether the box is compatible
with the VisionMate scanner. Every <em>position</em> in the box is identified
by a standard row letter + column number format (<em>e.g.</em>, C05).</p>

<hr />

<p>In this exercise, we are going to take tissues/samples/libraries from the
previous team/collaborator by scanning them out of the previous box and into a
newly created &#39;inbox&#39;.</p>

<p>First, create a new box. This box will be your &#39;inbox&#39;.</p>

<ol>
  <li>From the navigation panel, choose <em>Boxes</em> and then <em>Add Box</em>.</li>
  <li>On the <em>Create Box</em> page, enter the information:</li>
  <li>Alias: A short name for the box. Record this name in your worksheet. <img src="pics/blue_pencil.png" /></li>
  <li><em>Description</em>: a human description of the box purpose  “gDNA ready for
    library prep for <project>”.</project></li>
  <li><em>Use</em>: The contents of the box. Select an appropriate use for your entities.</li>
  <li><em>Size</em>: Select <code class="highlighter-rouge">8 x 12 scannable</code>.</li>
  <li>Click <em>Save</em>.</li>
</ol>

<p>Upon clicking save, a graphic of the box will appear.</p>

<p><img src="pics/5-empty-box.png" id="figure" /></p>

<p>You can now fill the box with the tissues, stocks, aliquots, libraries, or
pools that were made in previous steps. Collect the appropriate matrix tubes/barcodes now.</p>

<ol>
  <li>Click on a position in the displayed <em>Contents</em> grid.</li>
  <li>Enter a matrix barcode into the box on the right and click <em>Lookup</em>.</li>
  <li>Click <em>Update Position</em>.
    <ul>
      <li>&#9888; The <em>Save</em> button at the top of the page does not work for
 individual positions, only for <em>Box Information</em>.</li>
    </ul>
  </li>
  <li>Repeat for as many samples/libraries/pools as you want.</li>
</ol>

<p>The table below the box diagram shows the position and information for the
currently selected sample. If you would like to see all of the samples in the
table, click the <em>List all Box Contents</em> button at the top right of the table.</p>

<p>In the lab, it is possible to use <em>Options</em> &gt; <em>Scan Box</em> to use the plate
scanner and update all positions at once, but that will not be covered in this
tutorial.</p>

<h2 id="using-boxes">Using Boxes</h2>

<p>Boxes can be found either from the Sample or Library page or the <em>Boxes</em> page.</p>

<ol>
  <li>On the <em>Sample</em> page, search for one of your sample or library aliases and click on the alias to load the page.</li>
  <li>The Box and position is listed under <em>Location</em> near the top of the <em>Sample
Information</em> section. Click on the link to go to the Box (e.g. <code class="highlighter-rouge">PROJ_OUTBOX,
A03</code>).</li>
</ol>

<p>Boxes can be used to store Samples, Libraries and Pools and one box can store all
three types.</p>

<h2 id="bulk-editing">2.3 Bulk Editing</h2>

<p>Samples can be edited in bulk. Assume that we wish to update the description of the samples.</p>

<ol>
  <li>Select the sample you wish to edit:
    <ul>
      <li>If you are continuing from the end of section <strong>2.1 Receiving a Sample</strong>, do not
  navigate away from the page. At the top left of the table after saving samples, click
  the <em>Edit</em> button. Continue to step 2. Otherwise, select samples using the following:
        <ol>
          <li>On the <em>Samples</em> page, enter your project name in the search box.</li>
          <li>Check the boxes for the samples that you created in section 2.1.</li>
        </ol>
      </li>
    </ul>
  </li>
  <li>Click the <em>Edit</em> button at the top left of the table.</li>
  <li>Change the <em>Description</em> value: any value</li>
  <li>Click <em>Save</em>.</li>
</ol>

<h1 id="sample-qcs">3. Sample QCs</h1>

<p>Several QC methods are supported in MISO.</p>

<h2 id="adding-bulk-sample-qcs">3.1 Adding Bulk Sample QCs</h2>

<ol>
  <li>Select the samples you wish to edit:
    <ul>
      <li>If you are continuing from the end of section 2.3 do not navigate away from the page. At the top left of the table after saving samples, click the <em>Add QCs</em> button. Continue to step 2. Otherwise, select samples using the following:</li>
    </ul>
  </li>
  <li>On the <em>Samples</em> page, enter your project name in the search box.</li>
  <li>Check the boxes for the samples that you created in section 3.2.</li>
  <li>Click <em>Add QCs</em> at the top left of the table.</li>
  <li>Enter <code class="highlighter-rouge">1</code> QC per sample and click <em>Add</em>.</li>
  <li><em>Date</em>: enter today&#39;s date</li>
  <li><em>Type</em>: <code class="highlighter-rouge">Qubit</code></li>
  <li><em>Result</em>: <code class="highlighter-rouge">33</code></li>
  <li>Click <em>Save</em>.</li>
</ol>

<h1 id="scanning-samples-into-outboxes">4. Scanning samples into outboxes</h1>

<p>Finally, scan the samples you made into the outbox so the libraries team can take over.</p>

<p>Finally, update the <em>TUTORIAL_OUTBOX</em> with your samples/libraries/pools to
pass them on to the next team or to put them into storage.</p>

<h3 id="removing-and-discarding-tubes">Removing and discarding tubes</h3>

<p>When you have a position selected, you can also either remove the tube from the
box (setting its location to &quot;Unknown&quot;) or discard the tube, meaning it has been
used up. Discarding the tube sets the volume of the sample to 0 and marks it as
&quot;discarded&quot;, and removes it from its box.</p>

<ol>
  <li>Click on one of the positions in the box with a tube. The position, alias,
and barcode will appear to the right or below the box graphic. 
Right-click on the alias and pick &quot;Open in new tab&quot; to load the details page.</li>
  <li>Go back to the Box page in the other tab. Make sure the tube is still selected in
the Box and click <em>Discard Tube</em>. Click &quot;OK&quot; in the pop-up.</li>
  <li>Go to the other tab with the Details page and click the browser refresh
button. The <em>Location</em> field will show as blank, <em>Volume</em> will be set to 0.0,
and the <em>Discarded</em> box will be ticked.</li>
</ol>

<h3 id="moving-items-around-in-boxes">Moving items around in boxes</h3>

<p>An item can only exist in one box. If assigned to a new box, it will
disappear from the original.</p>

<ol>
  <li>In the current tab, go to the <em>Boxes</em> page, find the <em>TUTORIAL_OUTBOX</em> box. This box was previously
created by the MISO developers for the tutorial.</li>
  <li>Choose an empty position and enter one of your barcodes.</li>
  <li>Click <em>Lookup</em> and <em>Update Position</em>.</li>
  <li>Go back to your own Box and refresh the page. The sample was removed.</li>
  <li>Repeat this process for the rest of the samples/libraries that you are transferring to the next team.</li>
</ol>

<table>
  <tbody>
    <tr>
      <td>&lt; <a href="plain-1-0-project-coordination">Project coordination tutorial</a></td>
      <td><a href="plain-index">Home</a></td>
      <td><a href="plain-3-0-libraries">Libraries tutorial</a> &gt;</td>
    </tr>
  </tbody>
</table>



                    </div>
                
            </div>

            

            <div class="row">
                <div id="footer" class="col-sm-12">
                    Documentation for <a href="https://github.com/TGAC/miso-lims">MISO</a> developed by the <a href="http://labs.oicr.on.ca/genome-sequence-informatics/">Genome Sequence Informatics</a> group at <a href="http://oicr.on.ca">Ontario Institute for Cancer Research</a>


                </div>
            </div>
        </div>

        <script src="/miso-docs-oicr/styles/navigation.js"></script>

    </body>
</html>
