<!DOCTYPE html>
<html>
    <head>
        <meta charset="utf-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
        <meta name="viewport" content="width=device-width">

        <title>MISO : Libraries tutorial</title>
        <meta name="description" content="An open-source LIMS for NGS sequencing centres">

        <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css" rel="stylesheet">
        <link rel="stylesheet" href="/miso-docs-oicr/styles/main.css">
    </head>
    <body>

        <div class="container">
            <div class="row">
                <div id="header" class="col-sm-12">
                    <h2><a class="brand" href="/miso-docs-oicr/"><img src="/miso-docs-oicr/pics/MISO.png"/>MISO</a>
    <small>An open-source LIMS for NGS sequencing centres</small>
</h2>
<span class="badge" style="position: relative;left:25px;bottom:20px;background:rgb(56,121,59);">OICR</span>

<span><a href="/miso-docs-oicr/plain-index">See the original flavour (plain sample) version of this walkthrough here</a></span>


                </div>
            </div>

            <div class="row">
                
                
                    <div id="content" class="col-sm-12">
                        <div class="page-header">
    <h2>Libraries tutorial
        
    </h2>
</div>

<div id="toc">
Table of Contents
<ol>
   <li><a href="#login">Logging In</a></li>
   <li><a href="#props1">Propagating samples to libraries</a></li>
   <li><a href="#qcs">Adding Library QCs</a></li>
   <li><a href="#props2">Propagating libraries to dilutions</a></li>
   <li><a href="#boxes">Scanning libraries into your outbox</a></li>
   <li><a href="#pools">Creating Pools</a></li>
   <li><a href="#orders">Ordering sequencing</a></li>
   <li><a href="#trouble">Troubleshooting</a></li>
</ol>
</div>

<p><a name="login"></a></p>

<h1 id="logging-in">1. Logging in</h1>

<p>You need to log in to MISO LIMS in order to make changes to any LIMS entities. 
Logging in lets MISO record any changes you make on entities you have access to.
If you are a new user and have not been given a username and password, you 
will first need to get a username and password by following the steps in the
<a href="plain-0-0-admin-tasks#users">Adding Users</a> section.</p>

<hr />

<p>Try to log in now:</p>

<ol>
  <li>Click on <a href="http://miso.gsi.oicr.on.ca" target="\_new">http://miso.gsi.oicr.on.ca</a>.</li>
  <li>Enter your
username (e.g. jdoe) and password and click the Login button.</li>
</ol>

<p>If all goes well, you should see the MISO Dashboard and see a message at the
top right: &quot;Logged in as: <strong>jdoe</strong>&quot;.</p>

<p><a name="props1" href="#" id="toplink">top</a></p>

<h1 id="propagating-samples-to-libraries">2. Propagating samples to libraries</h1>

<p>A <em>library</em> is made from one sample for a single target <em>platform</em> and
has a specific <em>design</em> associated with it that decides the <em>selection</em>
and <em>strategies</em> used to make the library. A library may also have <em>indices</em>
(primers/sequencing barcodes/molecular IDs) and QC information.</p>

<h2 id="scan-samples-into-your-inbox">2.0 Scan samples into your inbox</h2>

<p>First, we accept the samples made by the samples team and put them into your
inbox.</p>

<p>Boxes hold samples, libraries, and pools. They do not track reagents or
primers. Boxes are separated into <em>uses</em> for different kinds of storage
(<em>e.g.</em>, tissue samples versus extracted DNA), but there is no check
that items in a box match. Each box also has a <em>size</em> that includes the
physical dimensions of the box as well as whether the box is compatible
with the VisionMate scanner. Every <em>position</em> in the box is identified
by a standard row letter + column number format (<em>e.g.</em>, C05).</p>

<hr />

<p>In this exercise, we are going to take tissues/samples/libraries from the
previous team/collaborator by scanning them out of the previous box and into a
newly created &#39;inbox&#39;.</p>

<p>First, create a new box. This box will be your &#39;inbox&#39;.</p>

<ol>
  <li>From the navigation panel, choose <em>Boxes</em> and then <em>Add Box</em>.</li>
  <li>On the <em>Create Box</em> page, enter the information:</li>
  <li>Alias: A short name for the box. Record this name in your worksheet. <img src="pics/blue_pencil.png" /></li>
  <li><em>Description</em>: a human description of the box purpose  “gDNA ready for
    library prep for <project>”.</project></li>
  <li><em>Use</em>: The contents of the box. Select an appropriate use for your entities.</li>
  <li><em>Size</em>: Select <code class="highlighter-rouge">8 x 12 scannable</code>.</li>
  <li>Click <em>Save</em>.</li>
</ol>

<p>Upon clicking save, a graphic of the box will appear.</p>

<p><img src="pics/5-empty-box.png" id="figure" /></p>

<p>You can now fill the box with the tissues, stocks, aliquots, libraries, or
pools that were made in previous steps. Collect the appropriate matrix tubes/barcodes now.</p>

<ol>
  <li>Click on a position in the displayed <em>Contents</em> grid.</li>
  <li>Enter a matrix barcode into the <em>Search</em> box on the right and click <em>Lookup</em>.
Normally a hand scanner would be used. You can also search for items by their
name or alias</li>
  <li>Click <em>Update Position</em>.
    <ul>
      <li>&#9888; The <em>Save</em> button at the top of the page does not work for
 individual positions, only for <em>Box Information</em>.</li>
    </ul>
  </li>
  <li>Repeat for as many samples/libraries/pools as you want.</li>
</ol>

<p>The table below the box diagram shows the position and information for the
currently selected sample. If you would like to see all of the samples in the
table, click the <em>List all Box Contents</em> button at the top right of the table.</p>

<p>In the lab, it is possible to use <em>Options</em> &gt; <em>Scan Box</em> to use the plate
scanner and update all positions at once, but that will not be covered in this
tutorial.</p>

<h2 id="using-boxes">Using Boxes</h2>

<p>Boxes can be found either from the Sample or Library page or the <em>Boxes</em> page.</p>

<ol>
  <li>On the <em>Sample</em> page, search for one of your sample or library aliases and click on the alias to load the page.</li>
  <li>The Box and position is listed under <em>Location</em> near the top of the <em>Sample
Information</em> section. Click on the link to go to the Box (e.g. <code class="highlighter-rouge">PROJ_OUTBOX,
A03</code>).</li>
</ol>

<p>Boxes can be used to store Samples, Libraries and Pools and one box can store all
three types.</p>

<h2 id="bulk-propagate-samples-into-libraries">2.1 Bulk propagate samples into libraries</h2>

<p>In this section, you will use the samples 
created already to create libraries.</p>

<p><img src="pics/plain-flow-aliquot.svg" /></p>

<ol>
  <li>
    <p>On the <em>Samples</em> page, enter your project name into the search box.</p>
  </li>
  <li>
    <p>Check the samples to turn into libraries.</p>
  </li>
  <li>Click the <em>Propagate</em> button at the top left of the table.</li>
  <li>Note that &quot;Propagate to libraries&quot; is the title of the box. Choose <em>1</em> replicate.</li>
  <li>Click <em>Propagate</em>.</li>
  <li>A table will appear. Enter the library information:
    <ul>
      <li><em>Library Name</em>: Leave blank as this will be filled in automatically after save.</li>
      <li>
        <p><em>Library Alias</em>: Leave blank as this will be filled in automatically.</p>
      </li>
      <li>
        <p><em>Description</em>: Library #, e.g. <code class="highlighter-rouge">Library 1</code></p>
      </li>
      <li><em>Platform</em>: Illumina</li>
      <li><em>Type</em>: Paired End</li>
      <li><em>Index Kit</em>: Nextera DNA Dual Index</li>
      <li><em>Index 1</em> and <em>Index 2</em>: Select any combination of indices you wish.
Select different indices for each library. Selecting the same index for two
different libraries will it unwise to pool those two libraries
together later.</li>
      <li><em>Kit</em>: Any kit</li>
      <li><em>Size (bp)</em>: 300</li>
    </ul>
  </li>
  <li>Choose <em>Save</em>.</li>
</ol>

<p>Note that for dual-index libraries, only the first index needs to be 
specified. The second is optional.</p>

<p><a name="receipt" href="#" id="toplink">top</a></p>

<h1 id="receiving-libraries">3. Receiving Libraries</h1>

<p>Propagating libraries from samples is only possible if there are existing samples to propagate from,
and only makes sense if the libraries were prepared in-house. Libraries may also be received without
the samples. The sample information is still required, but it can all be entered in one step.</p>

<h2 id="entering-received-libraries">3.1 Entering received Libraries</h2>

<ol>
  <li>On the left hand menu under <em>Tracking</em>, click <em>Libraries</em>.</li>
  <li>
    <p>Click the <em>Receive</em> button at the top of the table.</p>
  </li>
  <li>
    <p>In the dialog, select <em>Quantity</em>: 1, and click <em>Receive</em></p>
  </li>
  <li>Select or enter the following data:</li>
</ol>

<ul>
  <li>
    <p><em>Sample Alias</em>: enter a sample alias like <code class="highlighter-rouge">PRO1_S#_#</code>. Replace the <code class="highlighter-rouge">#</code> symbols with numbers of your choosing.</p>
  </li>
  <li><em>Sample Type</em>: select <code class="highlighter-rouge">GENOMIC</code> from the drop-down.</li>
  <li><em>Sci. Name</em>: Homo sapiens</li>
  <li>
    <p><em>Project</em>: Select your project from the drop-down.</p>
  </li>
  <li>
    <p><em>Date of receipt</em>: select today&#39;s date</p>
  </li>
  <li><em>Platform</em>: select Illumina from the drop-down</li>
  <li>
    <p><em>Type</em>: select Paired End from the drop-down</p>
  </li>
  <li><em>Selection</em>: PCR</li>
  <li>
    <p><em>Strategy</em>: WGS</p>
  </li>
  <li><em>Index Kit</em>: select No indices from the drop-down</li>
  <li><em>Kit</em>: select any option from the drop-down</li>
  <li><em>Size (bp)</em>: 430
    <ol>
      <li>Click <em>Save</em> at the top right</li>
    </ol>
  </li>
</ul>

<p><a name="qcs" href="#" id="toplink">top</a></p>

<h1 id="quality-control">4. Quality control</h1>
<p>There are three way to indicate library quality in MISO: 1)
Enter quantitative QC values under the <em>Library QC</em> section; 2) The overall 
pre-sequencing quality flag <em>QC passed</em>; and 3) The post-sequencing quality
control <em>Low quality library</em>.</p>

<h2 id="library-qc">4.1 Library QC</h2>
<p>After measuring the insert size or concentration, this information can be
entered into each library.</p>

<ol>
  <li>Select the libraries you wish to edit:
    <ul>
      <li>If you are continuing from the end of the section where you created libraries, do not 
navigate away from the page. At the top left of the table after saving libraries,
click the <em>Add QCs</em> button. Continue to step 2. Otherwise, select libraries using
the following:</li>
    </ul>
  </li>
  <li>On the <em>Libraries</em> page, enter your project name in the search box.</li>
  <li>Check the boxes for the libraries that you created previously.</li>
  <li>Click <em>Add QCs</em> at the top left of the table.</li>
  <li>Enter <code class="highlighter-rouge">1</code> QC per library and click <em>Add</em>.</li>
  <li><em>Date</em>: enter today&#39;s date</li>
  <li><em>Type</em>: <code class="highlighter-rouge">qPCR</code></li>
  <li><em>Result</em>: <code class="highlighter-rouge">22</code></li>
  <li>Click <em>Save</em>.</li>
</ol>

<h2 id="qc-passed">4.2 QC passed</h2>
<p><em>QC Passed</em> is a simple pass/fail flag for a library to decide if it is good
enough for sequencing. If not measured, this can be left as &quot;Unknown&quot;.</p>

<ol>
  <li>From the <em>Libraries</em> page, find one of your libraries using the search 
box and click the sample link.</li>
  <li>Change <em>QC passed</em> from <em>Unknown</em> to <em>True</em>.</li>
  <li>Click <em>Save</em>.</li>
</ol>

<h2 id="low-quality-sequencing">4.3 Low Quality Sequencing</h2>
<p>Libraries can be marked as having low sequencing quality, which will be shown
after the <em>Run</em> exercises.</p>

<p><a name="boxes" href="#" id="toplink">top</a></p>

<h1 id="scanning-libraries-into-your-outbox">5. Scanning libraries into your outbox</h1>

<p>Scan the libraries you just created into the outbox for the sequencing team to
use.</p>

<p>Finally, update the <em>TUTORIAL_OUTBOX</em> with your samples/libraries/pools to
pass them on to the next team or to put them into storage.</p>

<h3 id="removing-and-discarding-tubes">Removing and discarding tubes</h3>

<p>When you have a position selected, you can also either remove the tube from the
box (setting its location to &quot;Unknown&quot;) or discard the tube, meaning it has been
used up. Discarding the tube sets the volume of the sample to 0 and marks it as
&quot;discarded&quot;, and removes it from its box.</p>

<ol>
  <li>Click on one of the positions in the box with a tube. The position, alias,
and barcode will appear to the right or below the box graphic. 
Right-click on the alias and pick &quot;Open in new tab&quot; to load the details page.</li>
  <li>Go back to the Box page in the other tab. Make sure the tube is still selected in
the Box and click <em>Discard Tube</em>. Click &quot;OK&quot; in the pop-up.</li>
  <li>Go to the other tab with the Details page and click the browser refresh
button. The <em>Location</em> field will show as blank, <em>Volume</em> will be set to 0.0,
and the <em>Discarded</em> box will be ticked.</li>
</ol>

<h3 id="moving-items-around-in-boxes">Moving items around in boxes</h3>

<p>An item can only exist in one box. If assigned to a new box, it will
disappear from the original.</p>

<ol>
  <li>In the current tab, go to the <em>Boxes</em> page, find the <em>TUTORIAL_OUTBOX</em> box. This box was previously
created by the MISO developers for the tutorial.</li>
  <li>Choose an empty position and enter one of your barcodes.</li>
  <li>Click <em>Search</em> and <em>Update Position</em>.</li>
  <li>Go back to your own Box and refresh the page. The sample was removed.</li>
  <li>Repeat this process for the rest of the samples/libraries that you are transferring to the next team.</li>
</ol>

<p><a name="props2" href="#" id="toplink">top</a></p>

<h1 id="propagating-libraries-to-dilutions">6. Propagating libraries to dilutions</h1>

<p>A library cannot be directly loaded into a <em>lane</em> in a <em>sequencing container</em>
(flowcell/SMRTcell) in MISO. A dilution must be made and then many dilutions
(or just one) can be mixed into a <em>pool</em> for sequencing.</p>

<p><em>Orders</em> are requests for sequencing pools a certain number of times. They are
used to keep track of sequencing progress for project management and book-keeping.</p>

<h2 id="bulk-creating-library-dilutions">6.1 Bulk creating Library Dilutions</h2>
<p>Dilutions can be made in bulk from libraries.</p>

<p><img srg="pics/plain-flow-dilution.svg" /></p>

<p>In this exercise, we will create 4 library dilutions from the libraries we
made previously.</p>

<ol>
  <li>On the <em>Libraries</em> page, check all the libraries just created.</li>
  <li>From the toolbar, select <em>Make dilutions</em>.
    <ul>
      <li>Note that the <em>Make dilutions</em> option is also available after bulk creating
or bulk editing libraries.</li>
    </ul>
  </li>
  <li>Enter the dilution information:
    <ul>
      <li><em>Conc.</em>: (use any number you wish)</li>
      <li><em>Creation Date</em>: (use the current date)</li>
    </ul>
  </li>
  <li>Click <em>Save</em>.</li>
</ol>

<p><a name="pools" href="#" id="toplink">top</a></p>

<h1 id="creating-pools">7. Creating Pools</h1>

<p>Pools are the last step for libraries before sequencing and represent the
entity that is loaded onto the flowcell lane or SMRTcell. A &quot;pool&quot; can have one or more
library dilutions in it. Every lane of sequencing contains only one pool.</p>

<p><img src="pics/plain-flow-pool.svg" /></p>

<h2 id="bulk-creating-a-pool">7.1 Bulk creating a Pool</h2>

<p>Here we will pool all of the dilutions we added previously to make a single pool
of 4 dilutions.</p>

<ol>
  <li>Select the dilutions you wish to pool:
    <ul>
      <li>If you are still on the bulk dilutions page from section <strong>5.1</strong> do not navigate away from the page. Continue to step 2. Otherwise, select dilutions using the following:</li>
    </ul>
  </li>
  <li>On the <em>Dilutions</em> page, select the dilutions created.</li>
  <li>From the toolbar at the top left, click <em>Pool Together</em>.</li>
  <li>Enter the pool information:
    <ul>
      <li><em>Alias</em>: A short description of the pool contents. Enter the project name
followed by <code class="highlighter-rouge">_POOL</code> (e.g. <code class="highlighter-rouge">PROJ_POOL</code>)</li>
      <li><em>Description</em>: A longer free-text description of the pool.</li>
      <li><em>Creation Date</em>: Select a date.</li>
      <li><em>Concentration</em>: Enter a concentration.</li>
      <li><em>Volume</em>: Enter a pool volume.</li>
      <li><em>Ready to Run</em>: Whether or not the pool is ready for sequencing. This flag
  is used together with the Order to show the pool is ready to be sequenced.
  Make sure this is <em>True</em>.</li>
    </ul>
  </li>
  <li>Click <em>Save</em>.</li>
</ol>

<p><a name="orders" href="#" id="toplink">top</a></p>

<h1 id="ordering-sequencing">8. Ordering sequencing</h1>

<p>Orders are created on the pool to be sequenced, and include the quantity of sequencing
required (counted in lanes/SMRT cells), and the sequencing chemistry
required (on Illumina).</p>

<h2 id="creating-an-order">8.1 Creating an Order</h2>

<ol>
  <li>Select the pools you wish to create orders for:
    <ul>
      <li>If you are still on the bulk pools page from section <strong>6.1</strong> do not navigate away from the page. Continue to step 2. Otherwise, select pools using the following:</li>
    </ul>
  </li>
  <li>On the  <em>Pools</em> page, select the pool you just created.</li>
  <li>From the toolbar, click <em>Create Order</em>:</li>
  <li>Select the instrument for sequencing: <code class="highlighter-rouge">NextSeq 550</code></li>
  <li>Fill in the new order box:
    <ul>
      <li><em>Sequencing Parameters</em>: Select <code class="highlighter-rouge">High 2×151</code> chemistry.</li>
      <li><em>Lane Count</em>: the number of lanes that should run for this pool. Enter <code class="highlighter-rouge">2</code>.</li>
    </ul>
  </li>
  <li>Click <em>Save</em>.</li>
  <li>Click on the pool you just added an order to.</li>
  <li>The order will now be visible in the <em>Requested Orders</em> section.</li>
  <li>The outstanding sequencing required will be visible under <em>Order Status</em>.</li>
</ol>

<p>The language will vary depending on the platform (<em>i.e.</em>, PacBio containers
will show <em>SMRTcell Count</em> instead of <em>Lane Count</em>).</p>

<h2 id="checking-for-unfulfilled-orders">8.2  Checking for unfulfilled orders</h2>
<p>The <em>Orders</em> page is used to decide what needs to be sequenced.</p>

<ol>
  <li>From the navigation menu, choose <em>Orders</em>.</li>
  <li>Verify that the pool you just created is listed in the <em>Active</em> tab.</li>
</ol>

<p>Columns on this page will disappear if there are no entries (e.g.,
the <em>Failed</em> column will not be shown if there are no failed runs). When enough
lanes have been sequenced, the row will disappear from the <em>Active</em> tab,
but remain in the <em>All</em> tab. Lanes currently being sequenced will be marked as
in-progress and remain on the <em>Active</em> tab until the run transitions to
<em>Completed</em>.</p>

<p>A pool can have many orders. Orders for the same platform and chemistry are
summed when displayed on this page.</p>

<p><a name="trouble" href="#" id="toplink">top</a></p>

<h1 id="troubleshooting">Troubleshooting</h1>

<h2 id="how-do-i-correct-an-index-on-a-library">How do I correct an index on a library?</h2>
<p>The index can be changed on either the individual library page or the bulk edit page.</p>

<ol>
  <li>From <em>Tracking</em>, <em>Library</em>, select the library that needs to be changed.</li>
  <li>If necessary, change <em>Index Family</em>. If selected, the indices will be erased.</li>
  <li>Select the correct index under <em>Indices</em>. If the index family supports dual barcoding, another drop down will appear.</li>
</ol>

<h2 id="what-if-i-forget-to-put-a-library-dilution-in-a-pool">What if I forget to put a library dilution in a pool?</h2>
<p>If the library dilution has not been created:</p>

<ol>
  <li>From <em>Tracking</em>, <em>Library</em>, check the library that needs to be added.</li>
  <li>From the toolbar, click <em>Make Dilution</em>.</li>
  <li>Fill out the new row in the table, then click <em>Save</em>.</li>
</ol>

<p>Once the dilution exists:</p>

<ol>
  <li>From <em>Tracking</em>, <em>Pool</em>, select the pool that needs the additional library dilution.</li>
  <li>In the <em>Available Dilutions</em> section, enter the library name, library alias, or dilution name in the <em>Search</em> box.</li>
  <li>Find the correct dilution in the list and check it.</li>
  <li>Click <em>Add</em> on the toolbar.</li>
</ol>

<h2 id="how-do-i-make-bulk-orders">How do I make bulk orders?</h2>
<p>Making orders in bulk require the order be consistent for all pools.</p>

<ol>
  <li>From <em>Tracking</em>, <em>Pools</em>, select all the pools needing an order.</li>
  <li>Click <em>Create Order</em>, and enter the information as for a single order.</li>
</ol>

<p>A new order will be create for each pool with the same information.</p>

<table>
  <tbody>
    <tr>
      <td>&lt; <a href="plain-2-0-samples">Samples tutorial</a></td>
      <td><a href="plain-index">Home</a></td>
      <td><a href="plain-4-0-sequencing">Sequencing tutorial</a> &gt;</td>
    </tr>
  </tbody>
</table>



                    </div>
                
            </div>

            

            <div class="row">
                <div id="footer" class="col-sm-12">
                    Documentation for <a href="https://github.com/TGAC/miso-lims">MISO</a> developed by the <a href="http://labs.oicr.on.ca/genome-sequence-informatics/">Genome Sequence Informatics</a> group at <a href="http://oicr.on.ca">Ontario Institute for Cancer Research</a>


                </div>
            </div>
        </div>

        <script src="/miso-docs-oicr/styles/navigation.js"></script>

    </body>
</html>
